packages/components/src/components/option/option.stories.ts

```diff
@@ -34,149 +35,147 @@ export const Default = {
-      args
-    });
-  }
+export const Size = {
+  render: () =>
+    html` <div class="flex flex-row justify-between">
+      <sd-select
+        class="h-[260px] w-[220px]"
+        label="Large"
+        placeholder="Please Select"
+        placement="bottom"
+        max-options-visible="3"
+      >
+        <sd-option value="option-1" size="lg">Large Option 1</sd-option>
+        <sd-option value="option-2" size="lg">Large Option 2</sd-option>
+        <sd-option value="option-3" size="lg">Large Option 3</sd-option>
+      </sd-select>
+      <sd-select
+        class="h-[260px] w-[220px]"
+        label="Medium"
+        placeholder="Please Select"
+        placement="bottom"
+        max-options-visible="3"
+      >
+        <sd-option value="option-1" size="md">Medium Option 1</sd-option>
+        <sd-option value="option-2" size="md">Medium Option 2</sd-option>
+        <sd-option value="option-3" size="md">Medium Option 3</sd-option>
+      </sd-select>
+      <sd-select
+        class="h-[260px] w-[220px]"
+        label="Small"
+        placeholder="Please Select"
+        placement="bottom"
+        max-options-visible="3"
+      >
+        <sd-option value="option-1" size="sm">Small Option 1</sd-option>
+        <sd-option value="option-2" size="sm">Small Option 2</sd-option>
+        <sd-option value="option-3" size="sm">Small Option 3</sd-option>
+      </sd-select>
+    </div>`
 };
 
 /**
- * Shows available slots.
+ * Parent component `sd-select` controls state of `sd-option`.
  */
 
-export const Slots = {
-  parameters: {
-    controls: {
-      exclude: ['default', 'left', 'right']
-    }
-  },
-  render: (args: any) => {
-    return html`
-      ${['default', 'left', 'right'].map(slot =>
-        generateTemplate({
-          axis: {
-            x: {
-              type: 'slot',
-              name: slot,
-              title: 'slot=...',
-              values: [
-                {
-                  value:
-                    slot === 'default'
-                      ? `<div class="slot slot--border slot--background h-8 w-[100px]"></div>`
-                      : `<div slot='${slot}' class="slot slot--border slot--background h-6 ${
-                          slot === 'default' ? 'w-18' : 'w-6'
-                        }"></div>`,
-                  title: slot
-                }
-              ]
-            }
-          },
-          constants: [
-            {
-              type: 'slot',
-              name: 'left',
-              value: '<sd-icon slot="left" library="global-resources" name="system/picture"></sd-icon>'
-            },
-            {
-              type: 'slot',
-              name: 'right',
-              value: '<sd-icon slot="right" library="global-resources" name="system/picture"></sd-icon>'
-            }
-          ],
-          args
-        })
-      )}
-    `;
-  }
+export const Selected = {
+  render: () =>
+    html`<div class="flex flex-row gap-12">
+      <sd-select
```

Comment:
Pls show 3 options in each select and adjust text like in figma

![image](https://github.com/user-attachments/assets/fd5473eb-d1d4-409f-a24f-ac732ca5af75)